_sections/30-projects.md

```diff
@@ -15,3 +15,4 @@ style: center
 - [An Introduction to Applied Bioinformatics](http://readiab.org)
 - [QIIME](http://qiime.org)
 - [cual-id](https://github.com/johnchase/cual-id)
+- [PyStan](https://github.com/stan-dev/pystan]
```

Comment:
It should be a round paren `)` at the end. Also, I think you can move it up to just below scikit-bio - we're roughly ordering projects by prominence, and PyStan has more Github stars than IAB :-)